pom.xml

```diff
@@ -43,7 +43,7 @@
 
 		<!-- Depedency version properties -->
 		<version.httpclient>4.5.5</version.httpclient>
-		<version.openshift-client>4.10.1</version.openshift-client>
+		<version.openshift-client>4.9.2</version.openshift-client>
```

Comment:
Do we have confidence that this will not break anything?

Reply:
I will run level1 tests of our product openshift ts.

Reply:
Level1 tests have passed.